src/Models/LinkCheck.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Portable\FilaCms\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class LinkCheck extends Model
+{
+    use HasFactory;
+    use SoftDeletes;
+
+    protected $fillable = [
+        'title',
+        'origin_resource',
+        'edit_url',
+        'url',
+        'status_code',
+        'status_text',
+        'timeout',
+        'batch_id',
+    ];
+
+    public function latestBatch()
+    {
+        $batch = (new LinkCheck())->orderBy('created_at', 'DESC')->limit(1)->first();
```

Comment:
You should be able to just go like this:
`LinkCheck::orderby(...)` 
there is not need for the new etc.
if you need to you can do:
`LinkCheck::query()->orderBy(...)`